src/main/java/com/ecore/roles/web/dto/RoleDto.java

```diff
@@ -0,0 +1,46 @@
+package com.ecore.roles.web.dto;
+
+import com.ecore.roles.model.Role;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.validation.constraints.NotBlank;
+import java.util.UUID;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+@Builder
+@EqualsAndHashCode
+public class RoleDto {
+
+    @JsonProperty
```

Comment:
Add all the JSON properties names in the JsonProperty annotations, in order to don't get a code refactoring in the fields names leading to a mismatch in the API clients.